libs/api/domains/auth/src/lib/resolvers/delegationScope.resolver.ts

```diff
@@ -15,4 +18,18 @@ export class DelegationScopeResolver {
   resolveName(@Parent() delegationScope: DelegationScopeDTO): string {
     return delegationScope.scopeName
   }
+
+  @ResolveField('apiScope', () => ApiScope, { nullable: true })
+  resolveApiScope(
+    @Loader(ApiScopeLoader) apiScopeLoader: ApiScopeDataLoader,
+    @Parent() delegationScope: DelegationScope,
+    @Args('lang', { type: () => String, nullable: true, defaultValue: 'is' })
+    lang: string,
+  ): Promise<ApiScope | null> {
+    return apiScopeLoader.load({
+      lang,
+      domain: delegationScope.domainName,
```

Comment:
Same argument about domainName being unnecessary on apiScope, as the scopeName is unique primary key.

Reply:
We haven't planned a global scope REST resource. The API is domain scoped at the moment, and my instinct is to batch load them by domain instead of batch loading all scopes from all domains. Unless we create a custom batch load endpoint.

But this is basically implemented based on the current APIs and plans.